main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
since we know how many times to loop, maybe a for loop would be better for this situation. You could remove line 3 and 23 and minimizes the chance of creating infinite loop.